Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Debug/FingerCurlExample.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
```

Comment:
Please add a header here spelling out the license and such. See other classes in this project for a reference.